### PR TITLE
Add Dask Jobqueue documentation

### PR DESCRIPTION
**Pull request by holly-t**

### CODE REVIEW

File: docs/user-guide/python.md [hunk 1]
```diff
@@ -290,4 +290,65 @@ Please follow these steps.
...
 !!! note
     You willl of course need to use a different hostname for the SSH tunnel if you're running on the
-    ARCHER2 4-cabinet system, `login-4c.archer2.ac.uk`.   
\ No newline at end of file
+    ARCHER2 4-cabinet system, `login-4c.archer2.ac.uk`.   
+
+## Using Dask Job-Queue on ARCHER2
+
+The Dask-jobqueue project makes it easy to deploy Dask on ARCHER2. 
+You can find more information [here](http://jobqueue.dask.org/en/latest/).
```
Comment (aturner-epcc) [inline docs/user-guide/python.md:298]: Can you change "here" to "in the Dask Job-Queue documentation". The use of "here" for link text is problematic for accessibility reasons.

File: docs/user-guide/python.md [hunk 2]
```diff
@@ -290,4 +290,65 @@ Please follow these steps.
...
+
+
+cluster.scale(jobs=2)    # Deploy two single-node jobs
+
+from dask.distributed import Client
+client = Client(cluster)  # Connect this local process to remote workers
+
+# wait for jobs to arrive, depending on the queue, this may take some time
+import dask.array as da
+x = …
```
Comment (aturner-epcc) [inline docs/user-guide/python.md:345]: Is this an interactive Python script/session that runs on, e.g. the login/data analysis nodes or a script that gets submitted to the queues. I am probably being dense, but was not quite sure. Maybe a sentence clarifying where this runs would be helpful?

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Thanks @holly-t this looks really useful, just a couple of clarification comments

**Comment by holly-t** [comment]
@aturner-epcc I've made the change to the link text and added some further clarification.

**Review by aturner-epcc — APPROVED** [review]
Great thanks @holly-t I will merge this in now

**Pull request merged**